core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Get list of Helm repos 
+  become: yes
+  shell: |-
```

Comment:
Since `&&` is being used in the script, please use the `>-` here, this will remove extra new line characters.